packages/arbitrum/src/chain-ids.ts

```diff
@@ -0,0 +1,5 @@
+// The arbitrum Native Bridge only supports three chains, Arbitrum, Arbitrum Nova, and Ethereum
+export const CHAIN_ID_ARRAY = [1, 42161, 42170]
```

Comment:
any reason not to just use the named variables in the list? 

`export const CHAIN_ID_ARRAY = [ETH_CHAIN_ID, ARB_ONE_CHAIN_ID, ARB_NOVA_CHAIN_ID]`

Reply:
It should be - good catch